src/components/application_manager/rpc_plugins/vehicle_info_plugin/test/commands/hmi/vi_is_ready_request_test.cc

```diff
@@ -32,6 +32,7 @@
 
 #include "hmi/vi_is_ready_request.h"
 
+#include <vector>
```

Comment:
redundant include, `smart_object.h` has this one include

Reply:
@ydementieiev All required header files **must** be included explicitly